cmd/podman/push.go

```diff
@@ -76,17 +76,25 @@ var (
 )
 
 func pushCmd(c *cli.Context) error {
-	var registryCreds *types.DockerAuthConfig
+	var (
+		registryCreds *types.DockerAuthConfig
+		destName      string
+	)
 
 	args := c.Args()
-	if len(args) < 2 {
-		return errors.New("podman push requires exactly 2 arguments")
+	srcName := args[0]
+	if len(args) == 0 || len(args) > 2 {
+		return errors.New("podman push requires minimally a local image name and optionally a destination name")
```

Comment:
Isn't it the other way around? Minimally a destination name, and optionally a local image?

Reply:
I suppose it depends on how you look at it.  In the case where you provide only a singular arg, then that arg has to represent an image locally (or there is nothing to push).  What do others think?

Reply:
I agree with @edsantiago, it is a bit confusing. This reads like if you do `podman push alpine` it should push it to your registry, which is not the case.

Reply:
but what is the image is foobar.com/alpine:latest ?

Reply:
Maybe a local image tagged for push?